service/http.balancer/pom.xml

```diff
@@ -98,6 +98,24 @@
             <groupId>org.kaazing</groupId>
             <artifactId>k3po.junit</artifactId>
         </dependency>
+        <dependency>
```

Comment:
what does that do ?  I and aren't all these projects in git hub now ?  I  checked the other modules and I do not see an example of the dependency under this plug in ??